src/Illuminate/Validation/NotPwnedVerifier.php

```diff
@@ -15,15 +15,24 @@ class NotPwnedVerifier implements UncompromisedVerifier
      */
     protected $factory;
 
+    /**
+     * The number of seconds the request can run before timing out.
+     *
+     * @var int|null
```

Comment:
oh, actually, this one is fine as `int`. sorry.

Reply:
Oh yeah, you're right! 🤦 Fixed!